pkg/api/validation/validation.go

```diff
@@ -2044,6 +2055,9 @@ func ValidateLimitRange(limitRange *api.LimitRange) field.ErrorList {
 	for i := range limitRange.Spec.Limits {
 		idxPath := fldPath.Index(i)
 		limit := &limitRange.Spec.Limits[i]
+		if !validation.IsQualifiedName(string(limit.Type)) {
```

Comment:
Can you move this to a separate function and have it check for standard limit types and then require that things that are not standard resource types are qualified (i.e. they have a / ) similar to validateResourceName?